backend/src/apiserver/main.go

```diff
@@ -398,3 +408,21 @@ func initConfig() {
 
 	proxy.InitializeConfigWithEnv()
 }
+
+// Update getPVCSpec to also return a default size if present in the config
+func getPVCSpec() (*corev1.PersistentVolumeClaimSpec, string, error) {
+	workspaceConfig := viper.Sub("workspace")
```

Comment:
let's convert all the viper configs to constant values and put them to the top of the file, this includes `workspace`, `size`, and any other such config value (e.g. see `executionTypeEnv`)